requirements_ci.txt

```diff
@@ -0,0 +1,33 @@
+# pypi requirements for ci
+minio==7.0.1
```

Comment:
@georgosgeorgos not sure this should live in the root, I think it's best to relegate it into the .github folder since it's essentially CI related.